TweetBook/Program.cs

```diff
@@ -1,7 +1,11 @@
+﻿using TweetBook.Services;
+
 var builder = WebApplication.CreateBuilder(args);
```

Comment:
Is this how the tutorial did it in the Program.cs? What about the Startup file, is it empty?

Reply:
In .Net 6 we don't have the startup file, my bad. Maybe in the future we will organise better this file, but for now this conversation is resolved.